test/integration/api.bats

```diff
@@ -86,9 +86,41 @@ function teardown() {
 	[[ "${lines[3]}" == *"Nodes: 3" ]]
 }
 
-# FIXME
 @test "docker inspect" {
-	skip
+	start_docker 3
```

Comment:
yes very good idea!

ew should look for a `  "Node": {` object and a maybe `"Name": "node-"` as all the node start with `node-`